auth-web/src/components/auth/common/PaymentMethods.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div>
+    <v-card
+      outlined
+      hover
+      class="payment-card py-8 px-8 mb-4 elevation-1"
+      :class="{'selected': isPaymentSelected(payment)}"
+      v-for="payment in allowedPaymentMethods"
+      :key="payment.type"
+      @click="selectPayment(payment)"
+    >
+      <div class="d-flex">
+        <div class="mt-n2 mr-8">
+          <v-icon x-large color="primary">{{payment.icon}}</v-icon>
+        </div>
+
+        <div>
+          <div>
+            <div class="title font-weight-bold mt-n2 payment-title">{{payment.title}}</div>
+            <div>{{payment.subtitle}}</div>
+          </div>
+          <v-expand-transition>
+            <div v-if="isPaymentSelected(payment)">
+              <div class="pt-6">
+                <v-divider class="mb-6"></v-divider>
+                <div v-if="(payment.type === paymentTypes.PAD)">
+                  <!-- showing PAD form for PAD selection -->
+                  <PADInfoForm
+                    :padInformation="{}"
+                    @emit-pre-auth-debit-info="getPADInfo"
+                  ></PADInfoForm>
+                </div>
+                <div v-else
+                  v-html="payment.description">
+                </div>
+              </div>
+            </div>
+          </v-expand-transition>
+        </div>
+
+        <div class="ml-auto pl-8">
+        <v-btn
+          depressed
+          color="primary"
+          width="120"
+          class="font-weight-bold"
+          :outlined="!isPaymentSelected(payment)"
+          @click="selectPayment(payment)"
+          :aria-label="'Select' + ' ' + payment.title"
+        >
+          <span>{{(isPaymentSelected(payment)) ? 'SELECTED' : 'SELECT'}}</span>
+        </v-btn>
+        </div>
+      </div>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import { Account, PaymentTypes } from '@/util/constants'
+import { Component, Emit, Mixins, Prop, Vue } from 'vue-property-decorator'
+import PADInfoForm from '@/components/auth/common/PADInfoForm.vue'
+
+const PAYMENT_METHODS = {
```

Comment:
same for this..may be another common place

Reply:
the scope of this only applies to this component now. and I am planning to reuse the same component in the account settings as well. will disucss.